.pre-commit-config.yaml

```diff
@@ -18,3 +18,14 @@ repos:
   - id: end-of-file-fixer
   - id: check-yaml
   - id: check-added-large-files
+
+- repo: local
+  hooks:
+  - id: generate-manpage
+    name: generate manpage
+    language: python
+    additional_dependencies:
+      - argparse_manpage
```

Comment:
Let's add a similar line above to the `pylint` hook so it can check code in `man/build.py` as well.

Reply:
Done.